modules/text/src/precomp.hpp

```diff
@@ -48,6 +48,9 @@
 #include "text_config.hpp"
 
 #ifdef HAVE_TESSERACT
+#if !defined(USE_STD_NAMESPACE)
+#define USE_STD_NAMESPACE
```

Comment:
Macro is [here](https://github.com/tesseract-ocr/tesseract/blob/master/ccutil/platform.h#L57-L60) and only used by tesseract. May be insert in precomp.hpp : 
```
#include <string>
using std::string;
```